src/ResultWriter/faultoutput.f90

```diff
@@ -555,7 +555,7 @@ SUBROUTINE calc_FaultOutput( DynRup_output, DISC, EQN, MESH, MaterialVal, BND, t
 
               !add transient nucleation stress to fault output
               IF ((EQN%FL.EQ.103) .OR. (EQN%FL.EQ.3) .OR. (EQN%FL.EQ.4)) THEN
```

Comment:
Then if I understand correctly, you need to remove this if statement?
Else you won't see the initial stress on the fault output with other friction laws?

Reply:
You're right. I've removed it and checked tpv5 and 104, both work.